components/l10n/common/ofac_sanction_util_unittest.cc

```diff
@@ -62,6 +62,10 @@ struct ParamInfo final {
     {"en_SY", false, "en", "SY", true},
     {"en_SY", true, "en", "SY", true},
 
+    // Ukraine.
+    {"en_UK", false, "en", "UK", true},
```

Comment:
I believe that the 3166-2 code for Ukraine is UA, not UK, cf., https://en.wikipedia.org/wiki/ISO_3166-2

Reply:
Great catch, and the unit test just picked this up too :-)

Reply:
Fixed